openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -57,6 +58,11 @@ class TaxonomyView(ModelViewSet):
     """
     View to list, create, retrieve, update, delete, export or import Taxonomies.
 
+    TODO: We need to add a perform_udate and call the api update function when is created.
+    This is because it is necessary to call the model validations. (`full_clean()`).
+    Currently those validations are not run, which means we could update `export_id` with any value
+    through this api.
+
```

Comment:
I don't think we need to flag this risk here, because PUT requests are handled by `update_import`, which updates the tags.

We don't currently have a view for updating Taxonomy fields (but we can add one when needed).

Reply:
What I understand is that `update_import` is used with the url `/tagging/rest_api/v1/taxonomy/:pk/tags/import/plan` with PUT, but there is a view on `api/tagging/v1/taxonomy/:pk` with PUT to update the taxonomy.
I realized this because there are [tests for this view](https://github.com/openedx/openedx-learning/blob/2ec291845887b3db6bb56913ce1b12654d7e2574/tests/openedx_tagging/core/tagging/test_views.py#L413). I put this warning in case in the future the update of taxonomies is implemented in the UI

Reply:
Ok ya, I didn't realise that. But cool, thank you for noticing and adding that warning.